plugin-tester-scala/src/main/scala/io/grpc/examples/helloworld/GreeterServiceImpl.scala

```diff
@@ -1,5 +1,5 @@
 //#full-service-impl
-package io.grpc.examples
+package io.grpc.examples.helloworld
```

Comment:
Do we still want to move these files when we configure `flat_package` by default?

Reply:
Yes, if we want to have the same semantics as java files.

Reply:
I think that makes sense as a default